go.mod

```diff
@@ -4,7 +4,7 @@ go 1.23
 
 require (
 	github.com/notaryproject/notation-core-go v1.2.0-rc.1
-	github.com/notaryproject/notation-go v1.2.0-beta.1.0.20240926015724-84c2ec076201
+	github.com/notaryproject/notation-go v1.2.0-beta.1.0.20240927002524-3c4948808154
```

Comment:
We need to update the `notation-go` when `v1.3.0-rc.1` is released.

Reply:
Correct. We are 1 vote away from the release at the moment.

Reply:
Updated.